sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -17,17 +17,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(IsTargetingNetFx)' == 'true'">
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
-    <PackageReference Include="WindowsAzure.Storage" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(IsTargetingNetStandard)' == 'true'">
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" VersionOverride="11.0.0" />
```

Comment:
In this case, it looks as if this is the only project referencing the package.   If I'm not mistaken, we probably want to rev the main property file. 

@bsiegel:  Do you have a tool for scanning references?


![image](https://user-images.githubusercontent.com/913445/63546942-f3a32680-c4f8-11e9-955d-e1be0044a7e4.png)

Reply:
If you just want to see what projects use a given dependency, you can check the report here: https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/dependencies/dependencies.html

Instead of specifying the VersionOverride here you should update the version in the Packages.Data.props file. We only want to use VersionOverride if there is a specific reason we need to temporarily allow a library's dependency version to get out of sync with the rest of the SDK, and when that happens we need to file an issue to track eventual removal of the override.

Microsoft.Azure.EventHubs.Processor
--

Reply:
thanks!   I wasn't aware of that list.

Reply:
So as I understand it is safe to bump up the Blob version in Packages.Data.props. Can you confirm?

Reply:
Confirmed.   Please feel free to update `Packages.Data.props`

_(That was my suspicion after running a search on references and Brandon's feedback makes that conclusive.)_

Reply:
These cases ran OK on my desktop, but I see that a number of other cases have been marked "flaky" by the system. The common factor is slow startup, which is not a surprise in a testbed. I think this PR can go ahead -- these failures are a test issue, not a product issue. I will do a separate PR to increase the retry count.

Reply:
Unfortunately, these tests are failing CI and not just causing instability in the nightly runs.   We'll need to either mark them as ignore (and associate with an issue) or fix them before this can be closed out.

Reply:
Let me try running them on my desktop as well and update you.

Reply:
Latest build passed all tests. I will merge the PR since those tests are also flaky in master branch as well. James will fix them in a separate PR.